components/ResultsPage/YourAnswers.tsx

```diff
@@ -66,11 +67,15 @@ export const YourAnswers: React.VFC<{
         return String(Math.floor(Number(input.value)))
       case FieldType.DROPDOWN:
       case FieldType.DROPDOWN_SEARCHABLE:
-      case FieldType.RADIO:
         if ('values' in fieldData)
           return fieldData.values.find((value) => value.key === input.value)
             .text
         throw new Error(`values not found for field: ${input.key}`)
+      case FieldType.RADIO:
+        if ('values' in fieldData)
+          return fieldData.values.find((value) => value.key === input.value)
```

Comment:
The problem seems to be that TS isn't sure if the field is going to be a dropdown or a radio. Check out the screenshot below:

<img width="725" alt="Screen Shot 2022-07-26 at 9 54 04 PM" src="https://user-images.githubusercontent.com/17856371/181164396-0b037cfc-f214-49e9-9643-91e72eadb6ae.png">

Reply:
The solution is to convince TS that is in fact going to be a "FieldConfigRadio" when in this block. I would have expected TS to do that since we're in the "radio" case block but if I try doing this explicitly it seems to work ok:

<img width="795" alt="Screen Shot 2022-07-26 at 9 57 13 PM" src="https://user-images.githubusercontent.com/17856371/181164743-ee5da5e6-2e9c-4495-a42a-440e5a89891b.png">

It doesn't seem like THE solution but perhaps it puts you on the right track? Interestingly enough, it doesn't work if I compare the declared variable "fieldType" to check if it's "radio". It has to be "fieldData.type"

Reply:
@MarcoGoC not sure if you get notified when I leave comments but pinging you to check out the above. Let me know what you think or what you find.

Reply:
Thanks Alex, I just saw the comments.